docs/content/en/docs/use-cases/hpa.md

```diff
@@ -4,18 +4,171 @@ description: Using the HorizontalPodAutoscaler
 weight: 20
 ---
 
-Use the Kubernetes Custom Metrics API
-to refer to `KeptnMetric` via the
-[Kubernetes HorizontalPodAutoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
-(HPA),
-as in the following example:
+## Scaling Workloads based on Keptn metrics
+
+Kubernetes provides us with a lot of built-in capabilities to ensure
+running enough replicas in order to meet the current demand of the Workloads.
+One of these is the
+[HorizontalPodAutoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
+(HPA).
+
+HPA can make use of the [Keptn Metrics](../getting-started/metrics.md) custom API
```

Comment:
here we could add a link to https://kubernetes.io/docs/reference/external-api/custom-metrics.v1beta2/ to highlight that we are integrating with the kubernetes api extenstion